client/lang/en.js

```diff
@@ -30,7 +30,6 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "CMSMAIN.SELECTONEPAGE": "Please select at least 1 page.",
     "CMSMAIN.URLSEGMENTVALIDATION": "URLs can only be made up of letters, digits and hyphens.",
     "CMSMAIN.WARNINGSAVEPAGESBEFOREADDING": "You have to save a page before adding children underneath it",
-    "CMSMain.Archive": "Are you sure you want to archive this page and all of its children pages?\n\nThis page and all of its children will be unpublished and sent to the archive.",
```

Comment:
We can't remove strings from master keys, it'll delete it from 3.x branch too, since transifex is used as a single source for both branches.